src/_guides/language/language-tour.md

```diff
@@ -2613,6 +2542,25 @@ Point.fromJson(Map<String, num> json)
 The right-hand side of an initializer does not have access to `this`.
 </div>
 
+During development, you can validate inputs by using `assert` in the
+initializer list.
+
+<?code-excerpt "misc/lib/language_tour/classes/point_alt.dart (initializer-list-with-assert)"?>
+{% prettify dart %}
+// Initializer list can use assert().
```

Comment:
I'd suggest dropping the code comment and use highlighting on the snippet: `replace="/assert\(.*?\)/[!$&!]/g"`